core/pom.xml

```diff
@@ -373,6 +373,10 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.apache.curator</groupId>
+      <artifactId>curator-test</artifactId>
```

Comment:
`org.apache.curator.test.TestingServer` is from this artifact. An embedded ZooKeeper server for testing.